src/components/UserContacts/UserContacts.tsx

```diff
@@ -20,6 +21,11 @@ const StyledServicesList = styled.div`
     margin-bottom: ${gapM};
 `;
 
+const StyledRow = styled.div`
+    display: grid;
+    grid-template-columns: 1fr 28px 28px;
```

Comment:
What is 28px?

Reply:
<img width="689" alt="ы" src="https://github.com/taskany-inc/crew/assets/81121562/0ebc2ec1-cdf7-4b7f-a57f-986590965716">

To be positioned after <PageSep />

Reply:
This column width is set so that <IconMoreVerticalOutline />  is located after <PageSep />
<img width="349" alt="Untitled4" src="https://github.com/taskany-inc/crew/assets/81121562/423d2c20-ec7f-45a7-93df-848ecd3b12bd">